composer.json

```diff
@@ -17,10 +17,10 @@
     },
     "require": {
         "php": ">=5.5",
-        "guzzlehttp/guzzle": "^5.3||^6.0.1",
-        "guzzlehttp/psr7": "^1.0",
-        "guzzlehttp/promises": "^1.0",
-        "mtdowling/jmespath.php": "^2.2"
+        "guzzlehttp/guzzle": ">=5.3|>=6.0.1,<7",
```

Comment:
Well, i can't remember the exact error off the top of my head. If you're interested try it out I guess.